Newsletters: add 347 (2025-03-28)

harding opened the pull request:
- [x] Lede, releases, topic entries @harding 
- [x] Stack Exchange @bitschmidty 
- [x] Merges @Gustavojfe

LarryRuane commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,182 @@
+  for temporarily using an _HTLC slot_ (one of the limited number of
+  concurrent allocations available in a channel for enforcing
+  [HTLCs][topic htlc]).  A _hold fee_ would be paid by any node that
+  delays settlement of an HTLC; the amount of this fee would scale with
```
```suggestion
  delays the settlement of an HTLC; the amount of this fee would scale with
```

LarryRuane commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,182 @@
+  [HTLCs][topic htlc]).  A _hold fee_ would be paid by any node that
+  delays settlement of an HTLC; the amount of this fee would scale with
+  the length of the delay up until the maximum amount was reached at the
+  time the HTLC expired.  His post and paper cites several prior
```
```suggestion
  time the HTLC expired.  His post and paper cite several prior
```

LarryRuane commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,182 @@
+
+  The proposed protocol builds on the ideas of Law's _offchain payment
+  resolution_ (OPR) protocol (see [Newsletter #329][news329 opr]), which
+  has channel co-owners each allocate 100% the amount of funds at stake
```
```suggestion
  has channel co-owners each allocate 100% of the amount of funds at stake
```

LarryRuane commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,182 @@
+  resolution_ (OPR) protocol (see [Newsletter #329][news329 opr]), which
+  has channel co-owners each allocate 100% the amount of funds at stake
+  (so 200% total) to a _burnable output_ that either of them can
+  unilaterally destroy.  The funds at stake in this case is the upfront
```
```suggestion
  unilaterally destroy.  The funds at stake in this case are the upfront
```

LarryRuane commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,182 @@
+  (so 200% total) to a _burnable output_ that either of them can
+  unilaterally destroy.  The funds at stake in this case is the upfront
+  fee plus the maximum hold fees.  If both parties are later satisfied
+  that the protocol has been followed correctly, e.g. that all fees were
```
```suggestion
  that the protocol has been followed correctly, e.g., that all fees were
```

LarryRuane commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,182 @@
+  continue using testnet3 in the testnet version of his popular wallet
+  for at least a year.  Olaoluwa Osuntokun [noted][osuntokun testnet3]
+  that testnet3 has recently been much more stable than testnet4.  He
+  very effectively illustrated his point by including screenshots of the
```
```suggestion
  effectively illustrated his point by including screenshots of the
```
(overly-convincing language)

harding commented:
Added lede, added topic links (not including SE section), and added currently available RCs.  Reviewed @Gustavojfe section (thanks!).

I'll be back around 01:00 UTC to review the SE section, add any additional topic entries, add any additional releases/RCs, and address any reviews.  Thanks everyone!

bitschmidty commented:
Added stack exchange segment (sorry for the delay!)

Also included a topic entry fixup

bitschmidty commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,293 @@
+  rule.  This rule, which only applies to testnet, allows a block to be
+  valid with minimum difficulty if its header time is 20 minutes later
+  than its parent block.  Provoost goes into more [detail][provoost
+  testnet4] about the problem.  Poinsont proposes a testnet4 hard fork
```
```suggestion
  testnet4] about the problem.  Poinsot proposes a testnet4 hard fork
```

bitschmidty commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,293 @@
+  actually limited to 4,296 characters, but the 7,089 value is chosen
+  for LDK because "system-wide consistency is probably more important."
+
+- [LND #8453][], [#9559][lnd #9559], [#9575][lnd #9575], [#9568][lnd
```
8453 was closed? @harding @Gustavojfe

harding replied:
Yeah, but it was where the work originally occurred, so I think it's worth mentioning.  I listed the PRs in numeric order, so it (somewhat unfortunately) comes first.

azuchi commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,293 @@
+- [How long does it take for a transaction to propagate through the network?]({{bse}}125776)
+  Sr_gi points out that a single node cannot measure network-wide transaction
+  propagation times and that multiple nodes throughout the Bitcoin network would
+  be required to measure and estimate propogation times. He points to a
```
```suggestion
  be required to measure and estimate propagation times. He points to a
```

azuchi commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,293 @@
+  be required to measure and estimate propogation times. He points to a
+  [website][dsn kit] run by the Decentralized Systems and Network Services
+  Research Group at KIT that measures, among other things, transaction
+  propogation times that shows "a transaction takes about ~7 seconds to reach
```
```suggestion
  propagation times that shows "a transaction takes about ~7 seconds to reach
```

azuchi commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,293 @@
+
+- [How is the refund transaction in Asymmetric Revocable Commitments constructed?]({{bse}}125905)
+  Biel Castellarnau walks through the examples of commitment transactions from the
+  Master Bitcoin book.
```
```suggestion
  Mastering Bitcoin book.
```

LarryRuane commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,293 @@
+
+- [Bech32 error detection and correction reference implementation]({{bse}}125961)
+  Pieter Wuille notes that bech32 can detect up to 4 errors in address encoding
+  and can correct 2 substitution errors.
```
```suggestion
  and correct two substitution errors.
```

LarryRuane commented on _posts/en/newsletters/2025-03-28-newsletter.md:
```diff
@@ -0,0 +1,293 @@
+  and can correct 2 substitution errors.
+
+- [How to safely spend/burn dust?]({{bse}}125702)
+  Murch lists things to think about when trying to send [dust][topic
```
```suggestion
  Murch lists things to consider when sending [dust][topic
```
